docs/manual/commands/keybindings.rst

```diff
@@ -0,0 +1,37 @@
+=====================
+Keybindings in images
+=====================
+
+Default configuration
+=====================
+
+.. don't delete LS_PNG and END_LS_PNG (it is used for `make genkeyimg`)
+.. LS_PNG
+.. image:: /_static/keybindings/mod4__.png
+.. image:: /_static/keybindings/mod4-shift__.png
+.. image:: /_static/keybindings/mod4-control__.png
```

Comment:
Why do the files end with __?

Reply:
![__](https://user-images.githubusercontent.com/4191548/80481476-2be9e700-895b-11ea-8910-c5f2dcc576b5.png)
The name is made from modifiers, so if no modifiers - script should add something

Reply:
Let's add something like

```
if not modifier:
    filename = "base.png"
else:
    filename = "{}.png".format(modifier)
```

:)

Reply:
refactored to `no_modifier.png` and `modifier + '.png'`